src/components/collection/BindingSearch.tsx

```diff
@@ -218,81 +180,27 @@ function CollectionPicker({ readOnly = false }: Props) {
-                inputValue={inputValue}
-                size="small"
-                fullWidth
-                onChange={handlers.updateCollections}
-                onInputChange={(_event, newInputValue, reason) => {
-                    const inputBeingReset =
-                        detectAutoCompleteInputReset(reason);
-
-                    if (!inputBeingReset) {
-                        setInputValue(newInputValue);
-                    }
-                }}
-                blurOnSelect={false}
-                disableCloseOnSelect
-                disableClearable
-                renderTags={() => null}
-                renderInput={(params) => (
-                    <TextField
-                        {...params}
-                        label={collectionsLabel}
-                        required
-                        error={missingInput}
-                        variant="standard"
-                        onBlur={handlers.validateSelection}
-                    />
-                )}
-                renderOption={(props, option, { selected }) => {
-                    return (
-                        // TODO (styling) weirdly the paddingLeft was getting overwritten
-                        //  for dark mode and caused the icon to be too close to the edge
-                        //  so hardcoding the padding here for now
-                        <li {...props} style={{ paddingLeft: 24 }}>
-                            <Box
-                                sx={{
-                                    ml: -2,
-                                    mr: 0.5,
-                                }}
-                            >
-                                <Check
-                                    aria-checked={selected}
-                                    style={{
-                                        visibility: selected
-                                            ? 'visible'
-                                            : 'hidden',
-                                    }}
-                                />
-                            </Box>
-                            <Typography
-                                sx={{
-                                    ...truncateTextSx,
-                                }}
-                            >
-                                {option.name}
-                            </Typography>
-                        </li>
-                    );
-                }}
-            />
-        </Box>
+            getValue={(option: CollectionData) => option.name}
+            AutocompleteProps={{
+                getOptionLabel: (option: CollectionData) => option.name,
+                groupBy: (option: CollectionData) => option.classification,
+                componentsProps: {
+                    paper: {
+                        sx: {
+                            width: '33vw',
+                        },
+                    },
+                },
+            }}
+        />
```

Comment:
Now using the shared search. This file still exists to house the customizations that the binding selector needs.